azurelinuxagent/ga/exthandlers.py

```diff
@@ -423,6 +402,13 @@ def handle_ext_handler(self, ext_handler, etag):
 
         try:
             state = ext_handler.properties.state
+
```

Comment:
state = ext_handler.properties.state should come after this if statement

Reply:
Done